//
// This file is released under the terms of the NASA Open Source Agreement (NOSA)
// version 1.3 as detailed in the LICENSE file which accompanies this software.
//
//////////////////////////////////////////////////////////////////////

#ifndef MASSPROPSCREEN_H_
#define MASSPROPSCREEN_H_

#include "VspScreenQt.h"

class MassPropScreen : public VspScreenQt
{
public:
    MassPropScreen( ScreenMgr* mgr );
    ~MassPropScreen();
};

#endif /* MASSPROPSCREEN_H_ */
